### DISCUSSION TITLE
How to delete azure image if it not tagged in 24 hours from creation

### DISCUSSION THREAD

**Discussion by swati-delphix** [cloud-custodian/cloud-custodian#7432] [Azure]
### Ask your question

I want to delete an image if it is not tagged but sometimes image get tagged within 3 hrs by another job.
I have tried to use **marked-for-op**to delete in a day in azure , but it doesn't have unmark option.

**Comment by logachev** [comment]
@swati-delphix `mark-for-op` will work, but it does need 3 policies to work correctly:

1. Find untagged resources & mark for deletion
2. Find marked for deletion resources (using `value` filter to identify resources that have mark-for-op tag on them) and if they're properly tagged, remove `mark-for-op` tag
3. Find `mark-for-op` resources that are past due and delete them. This policy also has to check that resource is still untagged.

Please let me know if you need further help.

**Comment by swati-delphix** [comment]
@logachev So, we can tagged a resource but in documentation I didn't found unmark filter with the help of which we can remove mark-for-op tag. Would be help if you share example for same, as in AWS it is available but not in Azure.

**Comment by Bigstiffy1488** [comment]
Null
